CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 Changelog
 =========
 
+[1.2.3] - 2024-04-02
+--------------------
+
+### Other Changes
+
+- ci: Bump codecov/codecov-action from 3 to 4 (#193)
+
```

Comment:
```suggestion
```
Style for changelog entries is no spaces between entries - see elsewhere in this file